app/lib/pdf_filler/it201_additional_dependents_pdf.rb

```diff
@@ -0,0 +1,33 @@
+module PdfFiller
+  class It201AdditionalDependentsPdf
+    include PdfHelper
+    attr_accessor :dependent_offset, :intake, :source_pdf_name
+
+    def initialize(submission, dependent_offset: 7, source_pdf_name: "it201_additional_dependents")
+      @intake = submission.data_source
+      @dependent_offset = dependent_offset
+      @source_pdf_name = source_pdf_name
+    end
+
+    def hash_for_pdf
+      answers = {}
+      answers.merge!(dependents_info) if @intake.dependents.length > @dependent_offset
```

Comment:
in general i like it when we fill the pdf's from the xml, but i was having trouble with that myself for the dependent stuff so idk how much of a stickler i want to be about it...